components/engine_rocks/src/misc.rs

```diff
@@ -130,7 +130,12 @@ impl MiscExt for RocksEngine {
         for cf in self.cf_names() {
             handles.push(util::get_cf_handle(self.as_inner(), cf)?);
         }
-        self.as_inner().flush_cfs(&handles, wait).map_err(r2e)
+        let latest_seqno = self.get_latest_sequence_number();
```

Comment:
Why not call this inside `on_flush_completed`?

Reply:
Some CFs may not have any data then `on_flush_completed ` will never be called.

Reply:
Maybe you only need one cf to call `on_flush_completed`?

Reply:
How to distinguish flush from rocksdb and manual flush?